app/src/main/java/com/kickstarter/ui/extensions/ActivityExt.kt

```diff
@@ -71,6 +73,18 @@ fun Activity.showErrorSnackBar(anchor: View, message: String) {
     showSnackbarWithColor(anchor, message, backgroundColor, textColor)
 }
 
+fun Activity.setUpConnectivityStatusCheck(lifecycle: Lifecycle) {
+    val callBack = object : ConnectivityReceiver.ConnectivityReceiverListener {
+        override fun onNetworkConnectionChanged(isConnected: Boolean) {
+            if (!isConnected) {
+                showSnackbar(findViewById(android.R.id.content), getString(R.string.Youre_offline))
```

Comment:
FYI, For being able to use `setUpConnectivityStatusCheck(lifecycle)` on those activities migrated to compose we need to extend the extension with a callback specific for compose with `scaffoldState.snackbarHostState`, for now the callback is only valid for the `View` system.